src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -590,6 +618,9 @@ private List<LaunchCommand> GetInitializeCommands()
                 commands.Add(new LaunchCommand("-interpreter-exec console \"set pagination off\""));
             }
 
+            // When user specifies loading directives then the debugger cannot auto load symbols, the MIEngine must intervene at each solib-load event and make a determination
+            commands.Add(new LaunchCommand("-gdb-set auto-solib-add " + (_launchOptions.CanAutoLoadSymbols() ? "on" : "off")));
```

Comment:
>commands [](start = 12, length = 8)

Do you know if LLDB supports this?

Reply:
Since LLDB returns false from `SupportsStopOnDynamicLibLoad` seems like we shouldn't set this for LLDB.

---
In reply to: [216798249](https://github.com/Microsoft/MIEngine/pull/756#discussion_r216798249) [](ancestors = 216798249)

Reply:
Disabled the list for LLDB, so will either set this on or off for LLDB but not handle an exception list.